mobile/src/libs/enums/navigation/meditation-screen-name.enum.ts

```diff
@@ -0,0 +1,6 @@
+const MeditationScreenName = {
+  MEDITATION_TOPICS: 'Meditation topics',
```

Comment:
let's rename it to home, we won't have topics here
```suggestion
  MEDITATION_MENU: 'Meditation home',
```